pkg/repository/config/mongo_test.go

```diff
@@ -4,11 +4,11 @@ package config
 
 import (
 	"context"
+	"github.com/kubeshop/testkube/pkg/repository/storage"
```

Comment:
in go there is order in packages 

core 

local 

external 

why to break it ;)?

Reply:
my intelliJ configuration was wrong :/